src/Analysis/Engine/Test/AnalysisTest.cs

```diff
@@ -4068,7 +4091,7 @@ class x(object):
                 await server.OpenDefaultDocumentAndGetAnalysisAsync(code);
                 var signatures = await server.SendSignatureHelp(TestData.GetDefaultModuleUri(), signatureLine, 1);
 
-                signatures.Should().OnlyHaveSignature($"{expectedName}(a: complex, b: int, c: str, *d: {expectedDType})");
+                signatures.Should().OnlyHaveSignature($"{expectedName}(a, b, c, *d)");
```

Comment:
So we aren't going to show types in signatures, right?

Reply:
We show it if it is declared in the function definition (i.e. `def foo()`). We do not show 'derived' signatures that depend on actual passed parameters. Same as PyCharm.